### PR TITLE
adh branding

### PR DESCRIPTION
**Pull request by anton-dahlin**
This PR changes cosmetic branding from OSIsoft and OCS to AVEVA and ADH.

## What's Included
Following has been updated
* README link resources
* Company name
* OCS mentions
* Link titles
* Cloud portal links
* Sample library ref
* Resource in placeholder
 
The following is not included in this update
* History files
* GitHub links
* Build references
* Currently dead links such as ocs-docs (Will investigate best practice here) and OCS Overview pages (There is an option for this here https://www.aveva.com/content/dam/aveva/documents/datasheets/Datasheet_AVEVA_DataHubAnnouncementand_FAQ_10-21.pdf
* Copyright and License files

Let me know if anything is missing or incorrect above.

### CODE REVIEW

**Comment by anton-dahlin** [comment]
azp /run

**Review by derekendres — APPROVED** [review]

File: HISTORY.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
 # Version History
 
-## 1.3.0 / 2022-10-14
+## 1.3.1 / 2022-02-02
+
+- Updated dependencies
```
Comment (cbardini) [inline HISTORY.md:5]: Mention AVEVA rebranding

**Review by cbardini — APPROVED** [review]

**Pull request merged**